src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $color-white;
+  color: $main-color;
+
+  padding-inline: 16px;
+
+  @include hover(transform, scale(1.2));
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 0;
+  }
+
+  &__descr {
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    margin-top: 40px;
+    margin-bottom: 4px;
+
+   @include hover (color, #34568b)
+  }
+
+  &__code {
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    align-items: center;
+
+    &-title {
+     color: $secondary-color;
+     font-size: 12px;
+     line-height: 150%;
+    }
+
+    &-descr {
+      font-size: 16px;
+      line-height: 112.5%;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: inline-block;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    margin-bottom: 16px;
+    padding: 12px 70px;
+
+    border-radius: 5px;
+    background-color: $color-blue;
+
+    color: $color-white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    @include hover(color, $color-blue);
+    @include hover (background-color, $color-white);
+    @include hover (border, 1px solid $color-blue);
```

Comment:
This mixins will rewrite each other (transition property), please split the change transition logic.